extensions/wikia/CloseMyAccount/maintenance/CloseMyAccountMaintenance.php

```diff
@@ -2,6 +2,9 @@
 
```

Comment:
There are a lot of changes in this file because I replaced the single monolithic `execute` method with several smaller methods.